src/design-system/atoms/Button/Button.tsx

```diff
@@ -87,8 +91,8 @@ export const CustomButton: React.FC<ButtonProps> = ({
       size={size}
       startIcon={iconPosition === 'left' ? icon : null}
       endIcon={iconPosition === 'right' ? icon : null}
-      bordercolor={internalBorderColor}
-      texttype={lowercase ? 'none' : 'uppercase'}
+      borderColor={internalBorderColor}
```

Comment:
as discussed, these seem to be reserved words in mui and they will cause warnings, once this is switched back i will approve